recipe/build.sh

```diff
@@ -2,50 +2,16 @@
 
 set -x
 
-pushd pkg/CaratInterface
+pushd pkg/caratinterface
     tar pzxf carat.tgz
 popd
```

Comment:
This shouldn't be necessary, as CaratInterface does it automatically as part of its build process anyway
```suggestion
```

Reply:
Removed. Thanks